src/services/api/handlers/submit.ts

```diff
@@ -67,10 +70,10 @@ export const submit = async (event: APIGatewayEvent) => {
     }
 
     const today = seaToolFriendlyTimestamp();
-    const submissionDate = getNextBusinessDayTimestamp();
+    body.submissionDate = getNextBusinessDayTimestamp();
```

Comment:
ah, the difference is that, by adding submissionDate to the data object, it gets passed into the event and therefore into the email's "90th day" logic.  I see Ben says something about why overwrite submissionDate, so maybe it already IS in the event and I somehow missed it... in which case, I will revert :) 

Though, I have NEVER liked the requirement that the email sends the 90th day date, since the calculation of that date is full of hokeyness and putting it in the email somewhat turns the email into a source of truth for that date, when the formula can be changed for SEA Tool without the email being updated.  The person who insisted upon that date is gone, so perhaps we can ask to remove it.

Reply:
That would be FANTASTIC and solve something that has been worrying me for years now :)